src/climate_indices/__main__.py

```diff
@@ -90,6 +89,11 @@ def _validate_args(args):
                                          ("division", "time"),
                                          ("division")]
 
+    # dynamic threshold absolute tolerance parameter np.allclose
+    # derived from (smallest) grid size along dimension dim
+    def get_tolerance(dim):
+        return np.diff(dim).min() / 10
```

Comment:
This introduced a bug. If the latitudes are North to South, you get a negative absolute tolerance, and the `np.allclose()` will always fail.

Reply:
PR coming to fix this.